pisp_app/lib/Screens/account_dashboard.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/material.dart';
+import 'package:pispapp/MockData/account.dart';
+import 'package:pispapp/Screens/account_details.dart';
+
+class AccountDashboard extends StatefulWidget {
+  @override
+  _AccountDashboardState createState() => _AccountDashboardState();
+}
+
+class _AccountDashboardState extends State<AccountDashboard> {
+  List<Account> accounts = <Account>[];
+  List<String> androidVersionNames = <String>[];
+
+  @override
+  void initState() {
+    super.initState();
+
+    accounts = getMyDummyAccounts();
+    accounts = accounts.where((Account element) => element.linked).toList();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        body: Column(
+      children: <Widget>[
+        Expanded(
+          child: ListView.builder(
+            itemBuilder: (BuildContext context, int position) {
+              return InkWell(
+                onTap: () {
+                  Navigator.of(context).push<dynamic>(
+                    MaterialPageRoute<dynamic>(
+                      builder: (BuildContext context) => AccountDetails(accounts[position]),
+                    ),
+                  );
+                },
+                child: Column(
+                  children: <Widget>[
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: <Widget>[
+                        Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: <Widget>[
+                            Padding(
+                              padding: const EdgeInsets.fromLTRB(
+                                  12.0, 12.0, 12.0, 6.0),
+                              child: Text(
+                                accounts[position].alias,
+                                style: TextStyle(
+                                    fontSize: 22.0,
+                                    fontWeight: FontWeight.bold),
```

Comment:
```suggestion
                                    fontWeight: FontWeight.bold,),
```

By convention, I think it is usually preferred to [use trailing commas](https://flutter.dev/docs/development/tools/formatting#using-trailing-commas).

Reply:
I think this comment applies to a lot of other parts.